lib/src/authentication/src/client_options.dart

```diff
@@ -21,7 +21,15 @@ class ClientOptions extends AuthOptions {
 
   /// Custom log handler
   ///
+  /// For discussion about removing this component see
+  /// https://github.com/ably/ably-flutter/issues/238
+  ///
   /// https://docs.ably.com/client-lib-development-guide/features/#TO3c
+  @Deprecated(
+    'Not used, as log messages are handled by the default mechanism '
+    'in the underlying SDK. This instance variable will be removed '
+    'in future.',
```

Comment:
```suggestion
    'in a future release.',
```

Reply:
Fixed in https://github.com/ably/ably-flutter/pull/340/commits/a0d22cab50f7fa3781054cc118b78385f0fa3eea